api/src/movielist/movielist.service.ts

```diff
@@ -19,11 +19,12 @@ export class MovielistService {
 
   async cloneList(listId: string, userId: string, name?: string) {
     const originalList = await this.listService.getList(listId);
+    const originalMovies = await this.movieService.getMovies({ listId });
 
     const clonedList = await this.createListWithMovies(
       {
         name: name || originalList.name,
-        movie: originalList.movie,
+        movie: originalMovies,
```

Comment:
Considering that now we have pagination, wouldn't this `originalMovies` only have the first 10 movies of the list?